src/client/browserContext.ts

```diff
@@ -219,6 +219,18 @@ export class BrowserContext extends ChannelOwner<channels.BrowserContextChannel,
     return result;
   }
 
+  async setStorageState(options: SetStorageState) {
+    await this._wrapApiCall('browserContext.setStorageState', async () => {
+      await this._channel.setStorageState(options);
+    });
+  }
+
+  async storageState(): Promise<StorageState> {
```

Comment:
Since this is async function, I'd prefer a verb in the name, e.g. `saveStorageState` or `captureStorageSnapshot` or something.

Reply:
You mean like `page.screenshot()` or `page.pdf()`?

Reply:
I do have to say, this confused me in the [docs](https://playwright.dev/docs/auth#reuse-authentication-state):

```
// Save storage state into the file.
await context.storageState({ path: 'state.json' });
```

Especially since the description is `The following code snippet retrieves state from an authenticated context and creates a new context with that state.`.  I couldn't figure out why the result of `context.storageState`, which seemed to be getting storage state, wasn't returning anything that could be saved in the next step.  I'm a brand new playwright user, trying to figure out how I'd handle auth that includes MFA, which is how I ended up looking at this PR.